gobblin-runtime/src/main/java/org/apache/gobblin/runtime/GobblinMultiTaskAttempt.java

```diff
@@ -78,8 +78,7 @@
     /**
      * Commit {@link GobblinMultiTaskAttempt} immediately after running is done.
      */
-    IMMEDIATE,
-    /**
+    IMMEDIATE, /**
```

Comment:
Is this intentional? Shouldn't the "/**" start on a new line?

Reply:
I guess it's enforced based on the GobblinCode style done through "Reformat Code" action in  IntelliJ

Reply:
Just realized that adding one more empty line will fix it. Will make the change.